Gemfile

```diff
@@ -23,7 +22,8 @@ group :development do
   gem 'fakeweb'
   gem 'pry-doc'
   gem 'rspec',  '~> 3.0'
-  gem 'simplecov', '~> 0.7.1'
+  gem 'simplecov'
+  gem 'coveralls'
   gem 'yard'
 end
 
```

Comment:
I've read they recommend not requiring, e.g.
```
  gem 'simplecov', require: false
  gem 'coveralls', require: false
```
Could these be in a test group?  They're not required in develop, right?